src/main/java/ch/wisv/chue/HueService.java

```diff
@@ -155,6 +161,44 @@ public void alert(final int timeout) {
         new Thread(restoreStates, "ServiceThread").start();
     }
 
+    /**
+     * Strobes for some time
+     * @param millis duration
+     * @param lightIdentifiers which ligts
+     * @see <a href="http://www.lmeijer.nl/archives/225-Do-hue-want-a-strobe-up-there.html">Strobe with Hue by Leon Meijer</a>
+     */
+    public void strobe(int millis, String... lightIdentifiers) {
+        PHHueHttpConnection connection = new PHHueHttpConnection();
+        final String httpAddress = ((PHLocalBridgeDelegator)((PHBridgeImpl)bridge).getBridgeDelegator()).buildHttpAddress().toString();
+
+        // Put a light definition aka `symbol` at bulb, using internal API call
+        for (String light : lightIdentifiers) {
+            JSONObject pointSymbol = new JSONObject();
+            pointSymbol.put("1", "0A00F1F01F1F1001F1FF100000000001F2F");
```

Comment:
in "0A00F1F01F1F1001F1FF100000000001F2F"
staat de A (2e character) voor de strobe frequency, 4 is het laagst, F is het hoogst.